qa/tasks/ceph_manager.py

```diff
@@ -996,7 +996,7 @@ def wrapper(self):
                 raise
         return wrapper
 
-    @log_exc
+    @log_exc  # type: ignore
```

Comment:
which error do you get here?

Reply:
`error: Argument 1 to "log_exc" has incompatible type "Callable[[OSDThrasher], Any]"; expected "OSDThrasher"`

Reply:
Yeah, turns out, `log_exc` is a non-static method with the self-argument renamed to `func` and we're using the unbound `log_exc` method and passing the decorated function as the self-argument. 

So, we could either make `log_exc` a `staticmethod` and then use something like 

```python
class C:
     @staticmethod
     def deco(func):
         def w(self):
             func(self)
         return w

     @deco.__get__(object)
     def func(self):
         print(2)
```

or we can move log_exc to top-level. (my preferred solution)

Reply:
Done